LDM-294.tex

```diff
@@ -10,14 +10,15 @@
 \addtohist{3.0}{2017-06-30}{Complete overhaul of content, with all new authors. Rewritten in LaTeX. Approved for release by W.~O'Mullane.}{W.~O'Mullane}
 \addtohist{3.1}{2017-07-04}{Minor cleanups for review. Approved in \href{https://jira.lsstcorp.org/browse/RFC-358}{RFC-358}.}{W.~O'Mullane}
 \addtohist{3.2}{2017-07-19}{Editorial fixes and refresh schedule and component  diagrams}{W.~O'Mullane}
-\addtohist{3.3}{2017-08-21}{Update CCB and SE groups. Add Science Platform Scientist. Add section on design reviews. Approved in \jira{RFC-373}.}{W.~O'Mullane}
+\addtohist{3.3}{2017-08-21}{Update CCB and SE groups. Add Science Platform Scientist. Add section on design reviews. Approved in \href{https://jira.lsstcorp.org/browse/RFC-373}{RFC-373}.}{W.~O'Mullane}
+\addtohist{3.4}{2017-12-21}{Org chart refresh - rules for review of epics \href{https://jira.lsstcorp.org/browse/DM-7501}{DM-7501}}{W.~O'Mullane}
```

Comment:
Technically we don't update the version number until a release is done. Leave it blank. Add the release number later.

Reply:
Use `\jira{DM-7501}` to reference a JIRA ticket.

Reply:
ok will fix